packages/react-reconciler/src/__tests__/ReactBatchedMode-test.internal.js

```diff
@@ -1,167 +0,0 @@
-let React;
-let ReactFeatureFlags;
-let ReactNoop;
-let Scheduler;
-let ReactCache;
-let Suspense;
-let TextResource;
-
```

Comment:
Do any of these tests pass in CM?

Reply:
No they do not, and I'm sure which should pass.